.pre-commit-hooks.yaml

```diff
@@ -1,7 +1,9 @@
 - id: sphinx-lint
-  name: Sphinx lint
+  name: Sphinx Lint
   description: 'Searches for common typos in sphinx-flavored rst files.'
   files: '\.rst$'
```

Comment:
Or should we get rid of `files` altogether here, and _just_ use `types` in the default config?

```suggestion
```

Reply:
Yes I think just `types` should cover it.